assets/out

```diff
@@ -35,7 +35,20 @@ if [ -n "${username}" ] && [ -n "${password}" ] && [ -n "${email}" ]; then
 fi
 
 repository=$(jq -r '.source.repository // ""' < $payload)
-tag=$(jq -r '.source.tag // "latest"' < $payload)
+tag=$(jq -r '.params.tag // ""' < $payload)
```

Comment:
I think we should still support .source.tag in addition to .params.tag (params should take priority if both are specified).

This is used for treating them like branches, which used to be our workflow in some cases (i.e. push to 'latest' vs. 'dev').